calico/operations/decommissioning-a-node.mdx

```diff
@@ -23,15 +23,15 @@ of the following reasons.
 ## Purpose of this page
 
 Provide guidance on how to remove a host that is part of a {{prodname}} cluster
-and clean up the associated [Node resource][node resource reference]
+and clean up the associated [node resource reference](../reference/resources/node.mdx)
 information.
 
 ## Prerequisites
 
 - Prior to removing any Node resource from the datastore the `{{nodecontainer}}`
   container should be stopped on the corresponding host and it should be
   ensured that it will not be restarted.
-- You must have [calicoctl configured][calicoctl setup] and operational to run
+- You must have [calicoctl setup](/operations/calicoctl/install.mdx) and operational to run
```

Comment:
Does this link build correctly? Since you're already in operations, I'd expect the path to be `calicoctl/install.mdx`.